data/sql/updates/pending_db_world/rev_1609274862007339479.sql

```diff
@@ -0,0 +1,55 @@
+(-12166, 1, 2, 0, 60, 1, 100, 0, 0, 0, 0, 0, 0, 41, 3000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Portal Controller - On Update - Despawn In 3000 ms (Phase 1)'),
+(-12168, 1, 0, 1, 70, 0, 100, 0, 2, 0, 0, 0, 0, 51, 0, 0, 0, 0, 0, 0, 10, 63582, 17886, 0, 0, 0, 0, 0, 0, 'Sunhawk Portal Controller - On Gameobject State Changed - Kill Target'),
+(-12168, 1, 1, 0, 61, 0, 100, 0, 2, 0, 0, 0, 0, 22, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Portal Controller - On Gameobject State Changed - Set Event Phase 1'),
+(-12168, 1, 2, 0, 60, 1, 100, 0, 0, 0, 0, 0, 0, 41, 3000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Portal Controller - On Update - Despawn In 3000 ms (Phase 1)'),
+(-12173, 1, 0, 1, 70, 0, 100, 0, 2, 0, 0, 0, 0, 51, 0, 0, 0, 0, 0, 0, 10, 63585, 17886, 0, 0, 0, 0, 0, 0, 'Sunhawk Portal Controller - On Gameobject State Changed - Kill Target'),
+(-12173, 1, 1, 0, 61, 0, 100, 0, 2, 0, 0, 0, 0, 22, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Portal Controller - On Gameobject State Changed - Set Event Phase 1'),
+(-12173, 1, 2, 0, 60, 1, 100, 0, 0, 0, 0, 0, 0, 41, 3000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Sunhawk Portal Controller - On Update - Despawn In 3000 ms (Phase 1)');
+
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 182026 AND (`source_type` = 1));
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(182026, 1, 0, 0, 60, 0, 100, 0, 2000, 2000, 2000, 2000, 0, 105, 16, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Sun Gate - On Update - Add Gameobject Flags Not Selectable'),
+(182026, 1, 1, 0, 60, 0, 100, 0, 2000, 2000, 2000, 2000, 0, 106, 16, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Sun Gate - On Update - Remove Gameobject Flags Not Selectable');
+
+UPDATE `creature` SET `position_x`=-2147.350098, `position_y`=-10741.099609, `position_z`=73.903397 WHERE `guid`=63583;
+UPDATE `creature` SET `position_x`=-2128.939941, `position_y`=-10726.000000, `position_z`=66.335800 WHERE `guid`=63584;
+UPDATE `creature` SET `position_x`=-2107.070068, `position_y`=-10703.000000, `position_z`=65.189400 WHERE `guid`=63585;
+UPDATE `creature` SET `position_x`=-2098.620117, `position_y`=-10688.500000, `position_z`=65.218102 WHERE `guid`=63582;
+
+UPDATE `creature` SET `spawntimesecs` = 33 WHERE `id` = 17886;
+UPDATE `gameobject` SET `spawntimesecs` = 30, `animprogress` = 100 WHERE `id` = 184850;
+UPDATE `gameobject_template` SET `Data3` = 3000 WHERE (`entry` = 184850);
+
+DELETE FROM `creature` WHERE `guid` IN (96886);
```

Comment:
and ID

Reply:
I don't need to delete all of them, i am adding a new spawn alongside the 3 already existent